src/frontend/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "tsc --noEmit && webpack",
     "build-dev": "yarn copy-iframe-resizer; yarn build",
-    "generate-l10n-template": "rip json2pot 'i18n/**/*.json' -o i18n/frontend.pot",
+    "extract-translations": "formatjs extract \"./**/*.ts*\" --out-file i18n/frontend.json --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format react-intl-transformer.js",
```

Comment:
When I run this I get a few hundred `warning Error: Debug Failure. Output generation failed`. However in the end it does generate the `.json` file. Did you run into this too?

Reply:
Yes I have these errors too. Every files (`ts|tsx`) not containing translation generate an error. There is an option to generate an empty array if there is no translation but I don't use it.

Reply:
Alright, this is annoying but it does not prevent the extraction from working. I guess they need to add a flag to ignore these as they're not really errors 🤷‍♂️.